src/bug_report.py

```diff
@@ -79,6 +79,16 @@ def update_image_names_label(self):
 
     def send(self, controller):
         try:
+            error_description = self.error_description.toPlainText()
+
+            if not error_description:
+                message_box = QMessageBox(parent=self)
+                message_box.setWindowTitle(controller.view.locale.SubmittingBugReport)
+                message_box.setText(controller.view.locale.EmptyDescription)
+                message_box.setIcon(QMessageBox.Critical)
+                message_box.exec_()
+                return
```

Comment:
do not do like this. Bugs should go in a separate PR. [ok this time if you fix everything fast and we merge it in a few days]